src/net/server/world/World.java

```diff
@@ -103,6 +106,7 @@ public World(int world, int flag, String eventmsg, int exprate, int droprate, in
         petsSchedule = TimerManager.getInstance().register(new PetFullnessWorker(this), 60 * 1000, 60 * 1000);
         mountsSchedule = TimerManager.getInstance().register(new MountTirednessWorker(this), 60 * 1000, 60 * 1000);
         charactersSchedule = TimerManager.getInstance().register(new CharacterAutosaverWorker(this), 60 * 60 * 1000, 60 * 60 * 1000);
+        bossEntriesSchedule = TimerManager.getInstance().register(new BossEntriesWorker(this), calculateDifferenceTillMidNight(), 60 * 1000 * 1440);
```

Comment:
Swap calculateDifferenceTillMidNight() and 60 * 1000 * 1440. As confusing as it is, src/server/TimerManager.java swaps the positions for delay and repeatTime from ScheduledThreadPoolExecutor.

Reply:
Oh noted!